ptclient/cts/spi/rem/db/vuex-orm/hierarchy.js

```diff
@@ -1,3 +1,5 @@
-export default rowStatus from '@/cts/core/crud/rowstatus.js'
+// export default rowStatus from '@/cts/core/crud/rowstatus.js'
+// export { default as rowStatus } from '@/cts/core/crud/rowstatus.js'
 
-export default rem from './rem.js'
+// export default rem from './rem.js'
+// export { default as rem } from './rem.js'
```

Comment:
Only one default module can be exported. In the case of two, they will need to exported as named modules. This is just for illustration purposes, since this file isn't used (from what I can tell).